src/modules/sdk/objections/service.ts

```diff
@@ -27,6 +30,23 @@ const OBJECTIONS_API_SINGLE_ATTACHMENT_URL =
     (companyNumber: string, objectionId: string, attachmentId: string): string =>
       OBJECTIONS_API_URL(companyNumber) + `/${objectionId}/attachments/${attachmentId}`;
 
+/**
+ * Get the eligibility for the given company.
+ *
+ * @param {string} companyNumber the company number
+ * @param {string} token the bearer security token to use to call the api
+ *
+ * @returns {boolean} the eligibility for the given company
+ * @throws {ApiError}
+ */
+export const getEligibility = async (companyNumber: string, token: string): Promise<boolean> => {
+  logger.debug(`getting eligibility for company number ${companyNumber}`);
```

Comment:
I would change this log message slightly, just so that it differs from the one in the service (which I see is exactly the same). Maybe add 'from the SDK' onto the end of it?